src/deepsparse/yolo/README.md

```diff
@@ -0,0 +1,169 @@
+
+### Exporting the ONNX File From the Contents of a Local Directory
+This pathway is relevant if you intend to deploy a model created using the [SparseML](https://github.com/neuralmagic/sparseml) library. 
+For more information refer to the [appropriate YOLOv5 integration documentation in SparseML](https://github.com/neuralmagic/sparseml/tree/main/src/sparseml/yolov5).
+
+After training your model with `SparseML`, locate the `.pt` file for the model you'd like to export and run the `SparseML` integrated YOLOv5 ONNX export script below.
+
+```bash
+sparseml.yolov5.export_onnx \
+    --weights path/to/your/model \
+    --dynamic \ #Allows for dynamic input shape
+```
+This creates `model.onnx` file, in the directory of your `weights` (e.g. `runs/train/weights/model.onnx`).
+
+####  SparseZoo Stub
+Alternatively, you can skip the process of the ONNX model export by using Neural Magic's [SparseZoo](https://sparsezoo.neuralmagic.com/). The SparseZoo contains pre-sparsified models and SparseZoo stubs enable you to reference any model on the SparseZoo in a convenient and predictable way.
+All of DeepSparse's pipelines and APIs can use a SparseZoo stub in place of a local folder. The Deployment APIs examples use SparseZoo stubs to highlight this pathway.
+## Deployment APIs
+
+DeepSparse provides both a Python Pipeline API and an out-of-the-box model server
+that can be used for end-to-end inference in either existing Python workflows or as an HTTP endpoint.
+Both options provide similar specifications for configurations and support annotation serving for all 
+YOLOv5 models.
+
+### Python Pipelines
+Pipelines are the default interface for running inference with the DeepSparse Engine.
+
+Once a model is obtained, either through SparseML training or directly from SparseZoo, `deepsparse.Pipeline` can be used to easily facilitate end-to-end inference and deployment of the sparsified transformers model.
+
+If no model is specified to the Pipeline for a given task, the Pipeline will automatically select a pruned and quantized model for the task from the SparseZoo that can be used for accelerated inference. Note that other models in the SparseZoo will have different tradeoffs between speed, size, and accuracy.
+
+### DeepSparse Server
+As an alternative to Python API, the DeepSparse Server allows you to serve ONNX models and pipelines in HTTP.
+Both configuring and making requests to the server follow the same parameters and schemas as the
+Pipelines enabling simple deployment.  Once launched, a `/docs` endpoint is created with full
+endpoint descriptions and support for making sample requests.
+
+An example of starting and requesting a DeepSparse Server for YOLOv5 is given below.
+
+#### Installation
+The Deepsparse Server requirements can be installed by specifying the `server` extra dependency when installing
+DeepSparse.
+
+```bash
+pip install deepsparse[yolo,server]
+```
+
+## Deployment Example
```

Comment:
Because it's the formal case in other places here, this paragraph refers to "pipelines" - to be clear, is it capital [P]ipelines and [P]ipeline (two references)?